packages/lex_service/src/lexicon/services/lexicon.service.ts

```diff
@@ -21,6 +21,18 @@ export class LexiconService {
     return lexicon;
   }
 
+  async update(id: string, updateData: Partial<Lexicon>): Promise<Lexicon | null> {
+    // Filter out undefined fields from updateData
+    const updateFields = Object.fromEntries(
```

Comment:
Changing over to the dto approach mentioned above, you should also be able to avoid needing this filtering

Reply:
removed the filtering because of the dto approach